.github/workflows/build-staging-android.yml

```diff
@@ -48,60 +48,62 @@ jobs:
       - name: Install node_modules
         if: steps.modules-cache.outputs.cache-hit != 'true'
         run: yarn install
-      - name: Bundle install, fastlane dependencies
-        run: bundle install
+      - uses: ruby/setup-ruby@v1
```

Comment:
Maybe add a name for GH Actions GUI? Something like:
`- name: Download ruby and run bundle install`